lib/folio_sync/archives_space/marc_exporter.rb

```diff
@@ -0,0 +1,79 @@
+module FolioSync
+  module ArchivesSpace
+    class MarcExporter
+      ONE_DAY_IN_SECONDS = 24 * 60 * 60
+      PAGE_SIZE = 200
+
+      def initialize
```

Comment:
No need to change this right now, but it might be useful one day to be able to pass in `logger` and `archives_space_client` parameters to this initializer:

- stdout logger for manually run rake task mode and maybe Rails.logger (or some other log file) when the process runs on a cron
- One archives_space_client for our ArchivesSpace instance and another one for Barnard's

Reply:
Great idea, I'll address this once we start working on adding Barnard's instance